homeassistant/components/bluesound/media_player.py

```diff
@@ -831,6 +842,36 @@ def shuffle(self):
         else:
             _LOGGER.error("Master not found %s", master_device)
 
+    @property
+    def device_state_attributes(self):
+        """List members in group."""
+        attributes = {}
+        if self._group_list:
+            attributes = {ATTR_BLUESOUND_GROUP: self._group_list}
+
+        attributes[ATTR_MASTER] = self._is_master
+
+        return attributes
+
+    def rebuild_bluesound_group(self):
+        """Rebuild the list of entities in speaker group."""
+        bluesound_group = []
+
+        device_group = self._group_name.split("+")
```

Comment:
Don't we need the guard for  `self._group_name is None` anymore?

Reply:
Probably wise to keep it in there. Must admit that I put it in initially because it looked like a common pattern. The place where it matters is in `device_state_attributes`; either empty list or None produce the same result don't they?!

Reply:
`None` doesn't support `.split`, so that would raise an error.

Reply:
Ai... yess... overlooked that... Now changed back. Thanks!